tools/SetupFlow/DevHome.SetupFlow/Views/SetupFlowPage.xaml

```diff
@@ -103,14 +104,38 @@
                     <!-- Workaround to show a tooltip on a disabled button
                          https://github.com/microsoft/microsoft-ui-xaml/issues/2262-->
                     <setupFlowBehaviors:SetupFlowNavigationBehavior.DefaultNextTemplate>
-                        <Grid ToolTipService.ToolTip="{x:Bind ViewModel.Orchestrator.CurrentPageViewModel.NextPageButtonToolTipText, Mode=OneWay}">
+                        <Grid>

```

Comment:
The following steps cause multiple tooltip to show on the UI:
- From the Machine Configuration page, click on E2E page
- Navigate to the Dashboard
- Navigate back to the Machine Configuration page
- Hover over the Next button

<img src="https://github.com/microsoft/devhome/assets/104940545/cf07dea7-0466-419f-9cdb-1b4840e0b774" Width="500" />